hypervisor/qemu/qemu_process.go

```diff
@@ -13,6 +14,68 @@ import (
 	"github.com/hyperhq/runv/hypervisor"
 )
 
+type QemuLogFile struct {
+	name   string
+	offset int64
```

Comment:
the first letter of name and offset should be upper case for json marshal/unmarshal